src/karmen_frontend/src/routes/organizations/add-printer.js

```diff
@@ -150,14 +163,19 @@ class AddPrinter extends React.Component {
                 )}
                 <FormInputs
                   definition={form}
+                  collapsibleDefinition={{
```

Comment:
I don't really like this solution as it's rather unflexible. What if I want my collapsible field in the middle of the form, not at the end of it? Can't we make this part of the already existing `definition` block instead?

Reply:
Sure, I agree this is a hotfix solutions for a current needs. We're actually don't have a more (complex) requirements. In fact, I would suggest any existing solution for a form composition, but this is a bigger task.

Reply:
No, not OK at all. Please do not introduce more mess in the code by such hotfixes. It will last and more people will probably start using it making it harder to get rid off. The sole purpose of CRs is to avoid junk code.

Reply:
Done.